template_config.yml

```diff
@@ -3,7 +3,7 @@
 
 additional_repos:
 - bindings: false
-  branch: master
+  branch: v0.3.2
```

Comment:
Why are we pinning? I'm just unfamiliar with what issue is driving the pin.

Reply:
Also why pin here instead of in the requirements file itself? We pin there to https://github.com/pulp/pulp_ansible/blob/master/requirements.txt#L2

Reply:
some tests broke because galaxy importer is making `requires_ansible` mandatory

Reply:
it is pinned because we need to change orionutils - https://github.com/peaqe/orion-utils
I'll start a PR

Reply:
https://github.com/peaqe/orion-utils/pull/2

Reply:
@fao89 ty for opening https://github.com/peaqe/orion-utils/pull/2  Ideally we want https://github.com/peaqe/orion-utils/pull/2 to merge first so that we don't have to include corresponding changes in this PR. Am I understanding this right?

Reply:
actually, I have a workaround: https://github.com/pulp/pulp_ansible/pull/589/commits/657d1fd9b78ce3b0fa3f021ee7624aa4f0d1357d
so we don't need that PR, the PR is more a nice to have than required

Reply:
I was asking on their slack and they said they will merge https://github.com/peaqe/orion-utils/pull/2 I'd like to not carry a workaround if they'll merge it today.

Reply:
I was told on slack it's been released as `orion-utils==0.1.6`. Can this PR be reworked to use that instead of 657d1fd?